core/engine.cpp

```diff
@@ -39,6 +39,7 @@
 void Engine::set_iterations_per_second(int p_ips) {
 
 	ips = p_ips;
+	ERR_FAIL_COND(p_ips <= 0);
```

Comment:
This should be before `ips = p_ips;`, as it will cause an early return and thus prevent setting `ips` to an invalid value.